lib/DriverTool/swift_parse_test_main.cpp

```diff
@@ -113,6 +134,67 @@ struct SwiftParserExecutor {
   }
 };
 
+static void appendToVector(void *declPtr, void *vecPtr) {
```

Comment:
Is this used? Only reference I see is in ParseDecl.cpp

Reply:
Ah, it's a leftover. I'm removing it in a followup